student-work/2022/Yael-Lachman/ReadMe.md

```diff
@@ -1 +1,38 @@
+Technical Communicator
```

Comment:
```suggestion
# Yael Lachman  
Technical Communicator
```

Reply:
Add your name, so no one needs to look and see who you are in the URL or repo name.